src/backend/api/meals.js

```diff
@@ -1,15 +1,97 @@
+const { response, request } = require("express");
 const express = require("express");
 const router = express.Router();
 const knex = require("../database");
 
 router.get("/", async (request, response) => {
   try {
-    // knex syntax for selecting things. Look up the documentation for knex for further info
+
+    
+    if('maxPrice' in request.query){
+      const maxPrice = request.query.maxPrice;
+      const highPriceMeals = await knex ('meal').where ('price', '<=', maxPrice);
```

Comment:
```suggestion
      const highPriceMeals = await knex ('meals').where ('price', '<=', maxPrice);
```
 I believe the database is called `meals` and not `meal`

Reply:
Hi Florin,
My Table name is meal and not meals. Can i stick to the same name without changing?